packages-ui/roosterjs-react/lib/ribbon/plugin/createRibbonPlugin.ts

```diff
@@ -15,12 +17,18 @@ class RibbonPluginImpl implements RibbonPlugin {
     private timer = 0;
     private formatState: FormatState | null = null;
     private uiUtilities: UIUtilities | null = null;
+    private allowInsertHotKey: Boolean = true;
```

Comment:
Instead of saving each feature setting here, we can just save the whole options object. So that if we add more features in options, no need to add more members here.

Reply:
I completely agree. I think this is a cleaner way of doing it if we need to add more features.

Reply:
I completely agree, I think this is a cleaner way of doing it if we need to add more features.